cmd/dump.go

```diff
@@ -56,33 +57,63 @@ var dumpCmd = &cobra.Command{
 
 		fmt.Println("> Fetching policies")
 		wantedPolicies , _ := cmd.Flags().GetStringSlice("policies")
+		wantedAPIs , _ := cmd.Flags().GetStringSlice("apis")
+
 
 		policies := []objects.Policy{}
+		apis := []objects.DBApiDefinition{}
 		var errPoliciesFetch error
+		var errApisFetch error
+
+
+		if len(wantedAPIs) > 0 {
+			for _, APIID := range wantedAPIs{
+				api :=  objects.DBApiDefinition{}
+				api.APIID = APIID
+				apis = append(apis, api)
+			}
+
+		}
+		if len(wantedPolicies) > 0  {
+			for _,wantedPolicy := range wantedPolicies{
+				if !bson.IsObjectIdHex(wantedPolicy){
+					fmt.Println("Invalid selected Policiy ID:",wantedPolicy,".")
+					return
+				}
+				pol := objects.Policy{
+					ID: wantedPolicy,
+					MID: bson.ObjectIdHex(wantedPolicy),
+				}
+				policies = append(policies,pol)
+			}
+		}
+
+
+		if len(wantedAPIs) == 0 && len(wantedPolicies) == 0 {
```

Comment:
This check looks fine and it's different from the previous ones that don't log anything, we can keep it.